### PR TITLE
Update lircmap.xml for remote vista_mce 1039

### PR DESCRIPTION
**Pull request by karim789**
The remote is defined in lircd.conf.mceusb .

### CODE REVIEW

**Comment by mkortstiege** [comment]
@sraue, @stefansaraev mind having a look?

**Comment by MartijnKaijser** [comment]
@sraue @stefansaraev  ping

File: system/Lircmap.xml [hunk 1]
```diff
@@ -595,4 +595,99 @@
 		<recordedtv>KEY_PVR</recordedtv> 
 		<liveradio>KEY_RADIO</liveradio> 
 	</remote>
+	
+	<remote device="vista_mce">
+		<play>Play</play>
+		<pause>Pause</pause>
+		<stop>Stop</stop>
+		<forward>Fwd</forward>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: Forward or FastForward you meant ?

File: system/Lircmap.xml [hunk 2]
```diff
@@ -595,4 +595,99 @@
...
+		<reverse>Rwd</reverse>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: Rewind

File: system/Lircmap.xml [hunk 3]
```diff
@@ -595,4 +595,99 @@
...
+		<left>Left</left>
+		<right>Right</right>
+		<up>Up</up>
+		<down>Down</down>
+		<select>Ok</select>
+		<pageplus>Chup</pageplus>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: ChannelUp

File: system/Lircmap.xml [hunk 4]
```diff
@@ -595,4 +595,99 @@
...
+		<pageminus>Chdown</pageminus>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: ChannelDown

File: system/Lircmap.xml [hunk 5]
```diff
@@ -595,4 +595,99 @@
...
+		<back>Back</back>
+		<menu>Dvdmenu</menu>
+		<title>Guide</title>
+		<info>More</info>
+		<skipplus>Skipfwd</skipplus>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: SkipNext ?

File: system/Lircmap.xml [hunk 6]
```diff
@@ -595,4 +595,99 @@
...
+		<skipminus>Skipback</skipminus>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: SkipPrevious

File: system/Lircmap.xml [hunk 7]
```diff
@@ -595,4 +595,99 @@
...
+		<display>####Aspect</display>
```
Comment (stefansaraev) [inline system/Lircmap.xml]: did you mean AspectRatio ? and I see some extra ###

**Comment by MartijnKaijser** [comment]
@karim789 please update

**Comment by MartijnKaijser** [comment]
@stefansaraev maybe you can pick this one up and PR it?

**Comment by stefansaraev** [comment]
I am not sure I want to :)

**Comment by MartijnKaijser** [comment]
needs updating. closing for now

**Pull request closed without merging**